konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/impl/data/JData.java

```diff
@@ -250,6 +307,36 @@ else if (data instanceof Data) {
         return instance;
     }
 
+    public static Data singletonList(@NonNull String key, @NonNull List<?> data,
+                                     @NonNull ValueType valueType) {
+        Data instance = new JData();
+        if (valueType.equals(ValueType.STRING)) {
```

Comment:
ValueType is an enum - use `if(valueType == ValueType.STRING)` etc here instead.

Reply:
Fixed.